partner_event/__manifest__.py

```diff
@@ -9,13 +9,13 @@
 
 {
     "name": "Link partner to events",
-    "version": "15.0.1.0.0",
+    "version": "16.0.1.0.0",
     "category": "Marketing",
     "author": "Tecnativa," "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/event",
     "development_status": "Production/Stable",
     "license": "AGPL-3",
-    "depends": ["event"],
+    "depends": ["base", "event", "website"],
```

Comment:
Why these new deps?

Reply:
Thank you for reviewing! It's as a result of some errors raised when executing the tests. Messages like the following occur:

```
odoo.addons.base.models.ir_qweb.QWebException: Error while render the template
Exception: Unallowed to fetch files from addon website
Template: web.report_layout
```
In retrospect, adding the "website" dependency fixes the error and allows the tests to run, and probably "base" is unnecessary.

I'm new to migrating addons, so if you have a better way to handle this I'd appreciate any input!